index.js

```diff
@@ -1280,7 +1283,11 @@ Auth0Lock.prototype.oncloseclick = function(e) {
  */
 
 Auth0Lock.prototype.onescpressed = function(e) {
-  if ((e.which == 27 || e.keycode == 27)) this.hide();
+  self = this;
```

Comment:
A few comments:

1. `self = this;` missing `var` on this line.
2. There may be times where the popupCallback is undefined, which means that invoking it will crash the script.
3. I'm not sure an error is appropiate here, since it's not an authentication error. We do have an event called `hidden` which is invoked when the Lock is hidden.